array-methods-task1.js

```diff
@@ -12,13 +12,17 @@
 // camelize("-webkit-transition") == 'WebkitTransition';
 
 // P.S. Hint: use split to split the string into an array, transform it and join back.
-function camelize(string) {
-  return string
-    .split("-")
-    .map((word, index) =>
-      index == 0 ? word : word[0].toUpperCase() + word.slice(1)
-    )
-    .join("");
+
+function camelize(str) {
+  let array = str.split("-");
+  for (let i = 1; i < array.length; i++) {
+    array[i] = array[i].charAt(0).toUpperCase() + array[i].slice(1);
+
+    // console.log(array);
+    str = array.join("");
+  }
+  return str;
```

Comment:
The current code executes the line `str = array.join("")` for each part of the array. That's unnecessary, you can just do it once at the end :)

```suggestion
  }
  
  str = array.join("");

  return str;
```

Reply:
And if you want to save a line:

```suggestion
  }
  
  return array.join("");
```